configure.ac

```diff
@@ -1207,6 +1207,30 @@ AC_ARG_ENABLE([doxygen],
 
 AC_MSG_RESULT([$HDF5_DOXYGEN])
 
+## Check if they would like to enable doxygen warnings as errors
+##
+
+## This needs to be exposed for the library info file.
+AC_SUBST([HDF5_DOXY_WARNINGS])
+
+## Default is to not build DOXYGEN
+HDF5_DOXY_WARNINGS=no
```

Comment:
I think you want DOXY_ERR here. Also, the comment is misleading (copy-paste error?).

Reply:
done